lib/file_validators/validators/file_content_type_validator.rb

```diff
@@ -22,6 +22,8 @@ def validate_each(record, attribute, value)
           validate_whitelist(record, attribute, content_type, allowed_types)
           validate_blacklist(record, attribute, content_type, forbidden_types)
         end
+      rescue JSON::ParserError
```

Comment:
This rescues the entire `validate_each` method. The only place where JSON::ParseError error can originate is `parse_values` method. So we need to catch the error around `parse_values` call rather than rescuing the entire `validate_each` method.

Reply:
No callother than `parse_values` can raise a JSON::ParserError, that's true, and catching only this very specific kind of exception there's no risk to accidentally silent any other one. I can change it if you want but on the other hand, it makes the method more readable to not have a nested `begin ... rescue ... end`. (See [this Rubocop rule](https://docs.rubocop.org/en/stable/cops_style/#styleredundantbegin))

Reply:
Rescue around `parse_value` will make it more explicit. Clarity is more important than readability.

Reply:
Done